packages/lesswrong/components/posts/BookmarksList.tsx

```diff
@@ -8,25 +7,16 @@ const BookmarksList = ({limit=null}: {
   limit?: number|null
 }) => {
   const currentUser = useCurrentUser();
-  const { PostsItem2, Loading } = Components
+  const { PostsItem2 } = Components
 
-  const { document: user, loading } = useSingle({
```

Comment:
I think I remember putting this here intentionally, partially to reduce size of the currentUser, but also because there was some issue with the list auto-updating correctly. Maybe it works anyways, not sure. Would want to make sure you test the case of adding a bookmark and seeing it show up, and removing a bookmark and seeing it removed immediately.